### DISCUSSION TITLE
Table of content while editing

### DISCUSSION THREAD

**Discussion by tomzx** [damionx7/Zettel-Notes-Documentation#85] [Feature Requests]
It would be great if we could swipe left or right and it would bring a panel which displays the current table of content of the file. Clicking on any item would navigate to the specific header.

**Comment by aamott** [comment]
Agreed, or the option to generate (and keep updated) one at the top of the file.

**Reply by damionx7** [reply]
You can already generate table of contents at top of note using [TOC] tag.

eg.

```
[TOC]

Note content
```

**Comment by damionx7** [comment]
I have implemented this feature via outline feature in right drawer. It lists all the headings of a note and user can navigate by clicking on the heading.
